frontend/.eslintrc.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
@andrewtavis On my IDE(pycharm) the word `module `gives the following error: 
ESLint: 'module' is not defined.(no-undef). Lets check with another IDE and if error still exist we can add the following code to top of file: 
/*eslint-disable no-undef*/
or we can discuss about better solutions